clients/drcachesim/tracer/output.cpp

```diff
@@ -138,7 +138,7 @@ reached_traced_instrs_threshold(void *drcontext)
     tracing_window.fetch_add(1, std::memory_order_release);
     // We delay creating a new ouput dir until tracing is enabled again, to avoid
     // an empty final dir.
-    DR_ASSERT(tracing_mode.load(std::memory_order_acquire) == BBDUP_MODE_TRACE);
+    DR_ASSERT(mode == BBDUP_MODE_TRACE);
```

Comment:
I had changed this function so that mode == L0_FILTER was also part of the assert then I changed it again which left the assignment. I have removed the assignment.